### PR TITLE
Better docker build process

### PR DESCRIPTION
**Pull request by mattrobenolt**
@mitsuhiko

---

This change is [Reviewable](https://reviewable.io/reviews/getsentry/symsynd/5)

### CODE REVIEW

File: docker-build.sh [hunk 1]
```diff
@@ -5,4 +5,5 @@ cd -P -- "$(dirname -- "$0")"
 
 mkdir -p dist
 docker build -t symsynd:dev .
-docker run --rm -v $PWD/llvm:/symsynd/llvm -v $PWD/dist:/symsynd/dist symsynd:dev "$@"
+docker run --rm symsynd:dev > tar -xC dist
+ls dist/*
```
Comment (mattrobenolt) [inline docker-build.sh]: Here's your `*.whl` outside the container without requiring volumes.

**Comment by mattrobenolt** [comment]
Should be good to go now.

**Comment by mitsuhiko** [comment]
Not a fan of removing the git submodule for llvm. Is there a specific reason you are fetching the archive? But other than that looks good. I will review this in detail.

**Comment by mitsuhiko** [comment]
This breaks the non docker build for what it's work `make develop` falls flat on the floor because nothing is set up to actually download llvm.

**Pull request merged**